### DISCUSSION TITLE
Syncthing support to sync current state in podcasts

### DISCUSSION THREAD

**Discussion by therealmaxmoon** [PaulWoitaschek/Voice#2422] [Ideas]
I am syncing all my podcasts between several devices. It is pretty comfortable to look for them on the pc and then just copy the ones I like to a specific folder and let Syncthing do the rest.

All those folders are synchronized with my pcs/mobile devices and I can listen to them whenever I want without copying them by hand.

But the problem is, the devices don't know were I left an episode of a podcast to I have to figure it out by myself or listen it all over again, which could be annoying.

Might it be possible to give every directory or even file another file with the current information were the user currently is with a timestamp or something similar?

This would be very useful and could be used on a pc to, because a simple script could read the timestamp open the music file with mpv and jump to the time the user left it on the mobile phone. And the script could write a new timestamp, if it mpv was closed before the mp3 ends. And this file would be synced back to all other devices. 

I am curious on how this is even possible or if there is even another solution to solve it?

**Comment by Penguinjumper** [comment]
This would be so amazing!!!
An easy solution would be to just save progress in small text files in the folders where the audiobooks are saved. "Smart Audiobook Player" does this.

**Reply by therealmaxmoon** [reply]
I am still waiting for this cool feature, because within a year I couldn't find anything close to this. And it would be sooo amazing to have a cloudless, decentralized feature to synchronize the current state between devices. 

Would love to start an mp3 with mpv on my linux desktop and automatically resume the podcast I've listened to on the phone.

I would even create a terminal app for linux to continue listening podcasts on the computer and write new timestamps to the file, if this android app would have this features. I am serious! There is not much I want in my life, but this is one of those things :)

**Reply by Penguinjumper** [reply]
After a lot of trying the only thing I found close to this (short of selfhosting a Plex/jellyfin server) is ibroadcast. Their service is pretty good for audiobooks and podcasts if you use the "one queue" and bookmarks-functions.

**Reply by mpetuska** [reply]
Progress sync with jellyfin would be awesome!
